components/dashboard/src/provider-utils.tsx

```diff
@@ -12,13 +12,13 @@ import { gitpodHostUrl } from "./service/service";
 function iconForAuthProvider(type: string) {
     switch (type) {
         case "GitHub":
-            return <img className="fill-current dark:filter-invert w-5 h-5 ml-3 mr-3 my-auto" src={github} />;
+            return <img className="fill-current dark:filter-invert w-5 h-5 ml-3 mr-3 my-auto" src={github} alt="" />;
```

Comment:
alt can be filled out too 😅 

`alt="GitHub"`

Reply:
Absolutely valid point, @Siddhant-K-code! Could you open a follow-up issue or PR for this?

Reply:
I've created this PR: https://github.com/gitpod-io/gitpod/pull/15617